Makefile

```diff
@@ -0,0 +1,8 @@
+VCS_REF="$(shell git rev-parse HEAD)"
+
+BUILD_DATE="$(shell date -u +"%Y-%m-%dT%H:%m:%SZ")" 
+
+docker-build:
+	docker build -t exo .
+	docker tag exo $(DOCKER_ID_USER)/exo
+	docker push $(DOCKER_ID_USER)/exo
```

Comment:
1. no push
2. you'll have to use the `--build-arg`
3. is the cli `main.version` ready?

Reply:
okay thx